content/applications/inventory_and_mrp/manufacturing/quality_control/quality_checks.rst

```diff
@@ -0,0 +1,77 @@
+==============
+Quality checks
+==============
+
+.. |QCP| replace:: :abbr:`QCP (Quality Control Point)`
+
+Quality checks are manual inspections conducted by employees, and are used to ensure the quality of
+products. A quality check can be conducted for a single product, or multiple products within the
+same inventory operation or manufacturing order.
+
+Using a Quality Control Point (QCP), it is possible to create quality checks automatically at
+regular intervals. A |QCP| can be configured to create quality checks for specific products or
+operations, or a combination of both. For a full explanation of how to create and configure a |QCP|,
+see the documentation on :ref:`Quality Control Points
+<manufacturing/quality_control/quality-control-points>`.
+
+While quality checks are most commonly created through a |QCP|, it is also possible to manually
+create a single quality check. Doing so may be necessary if quality checks need to be conducted
+outside of those created at regular intervals by a |QCP|.
+
+Manual quality check
+====================
+
+To manually create a single quality check, navigate to :menuselection:`Quality --> Quality Control
+--> Quality Checks`, and click :guilabel:`New`. On the quality check form, begin by selecting an
+option from the :guilabel:`Control per` drop-down menu:
+
+- :guilabel:`Operation` requests a check for an entire operation (ex. delivery order) and all
+  products within it.
+- :guilabel:`Product` requests a check for every unit of a product that is part of an operation (ex.
+  every unit of a product within a delivery order).
+- :guilabel:`Quantity` requests a check for every quantity of a product that is part of an operation
+  (ex. one check for five units of a product within a delivery order). Selecting :guilabel:`Quantity`
+  also causes a :guilabel:`Lot/Serial` drop-down field to appear, from which can be selected a
+  specific lot or serial number that the quality check should be conducted for.
+
+Next, select an inventory operation from the :guilabel:`Picking` drop-down menu or a manufacturing
+order from the :guilabel:`Production Order` drop-down menu. This is necessary because Odoo needs to
+know which operation the quality check is being conducted for.
```

Comment:
```suggestion
Next, select an inventory operation from the :guilabel:`Picking` drop-down menu, or a manufacturing
order from the :guilabel:`Production Order` drop-down menu. This is necessary because Odoo needs to
know which operation for which the quality check is being conducted.
```